openreview/venue_request/process/deployProcess.py

```diff
@@ -320,3 +323,63 @@ def process(client, note, invitation):
         },
         signatures=['~Super_User1']
     ))
+
+    if (forum.content.get('Submission Deadline', None) and forum.content.get('Paper Matching', None)):
```

Comment:
`None` is already the default value for get, no need to define it as a parameter.